config.json

```diff
@@ -18,5 +18,9 @@
   },
   "proxy": "",
   "AWS_ACCESS_KEY_ID": "XXXXXXXXXXXXXXXXX",
-  "AWS_SECRET_ACCESS_KEY": "XXXXXXXXXXXXXXXXXXXXX"
+  "AWS_SECRET_ACCESS_KEY": "XXXXXXXXXXXXXXXXXXXXX",
```

Comment:
@princu : We should add the info to update these keys in the readme.md. What you suggest ?